{"name": "test_adding_two_numbers", "status": "broken", "statusDetails": {"message": "pytest_bdd.exceptions.StepDefinitionNotFoundError: Step definition is not found: Given \"I have the number 5\". Line 7 in scenario \"Adding two numbers\" in the feature \"C:\\Users\\neete\\Downloads\\MinimalProject\\features\\my_feature.feature\"", "trace": "E   pytest_bdd.exceptions.StepDefinitionNotFoundError: Step definition is not found: Given \"I have the number 5\". Line 7 in scenario \"Adding two numbers\" in the feature \"C:\\Users\\neete\\Downloads\\MinimalProject\\features\\my_feature.feature\"\nAll traceback entries are hidden. Pass `--full-trace` to see hidden and internal frames."}, "description": "C:\\Users\\neete\\Downloads\\MinimalProject\\features\\my_feature.feature: Adding two numbers", "start": 1743507300393, "stop": 1743507300393, "uuid": "cc21a116-cd41-4239-bd4f-b7ff2177e1e9", "historyId": "8d35b6e7ffab4daff71518c22c74f3dd", "testCaseId": "8d35b6e7ffab4daff71518c22c74f3dd", "fullName": "tests.test_my_feature#test_adding_two_numbers", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_my_feature"}, {"name": "host", "value": "user"}, {"name": "thread", "value": "18576-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_my_feature"}]}